pkg/controller/installation/installation_controller_test.go

```diff
@@ -99,6 +99,64 @@ func testCreateInstallationCR_useClusterStorage(t *testing.T, envValue string, a
 	return nil
 }
 
+func TestCreateInstallationCR_alertingEmailAddressIsPresent(t *testing.T) {
+
+	mockClient := fake.NewFakeClientWithScheme(buildScheme())
+	ctx := context.TODO()
+
+	email := "noreply-test@rhmi-redhat.com"
+	os.Setenv("ALERTING_EMAIL_ADDRESS", email)
+	os.Setenv("WATCH_NAMESPACE", defaultNamespace)
+
+	// Defer unsetting the environment variables regardless of test results
+	defer os.Unsetenv("ALERTING_EMAIL_ADDRESS")
+	defer os.Unsetenv("WATCH_NAMESPACE")
+
+	// Function to test
+	err := createInstallationCR(ctx, mockClient)
+
+	if err != nil {
+		t.Fatalf("Error creating installation CR: %v", err)
+	}
+
+	installation, err := getInstallationCR(ctx, mockClient, t)
```

Comment:
Should `err` be handled? The test will panic if a `nil` cr is returned for whatever reason from this function

Reply:
Good catch, thanks